dist/.ebextensions/01_bridgecf.config

```diff
@@ -4,11 +4,12 @@ files:
     owner: root
     group: root
     content: |
-        #!/bin/env bash -x
+        #!/bin/bash
         # Script to dynamically build bridge server config from env variables
-        # Brian Holt <beholt@gmail.com> 02.09.2017
-        mkdir -p ${HOME}/.bridge
-        cat <<-EOF > ${HOME}/.bridge/bridge-server.conf
+        # Brian Holt <beholt@gmail.com> 03.30.2017
+        mkdir -p /home/webapp/.bridge
+        chown ${EXECUSER} /home/webapp/.bridge
+        cat <<-EOF > /home/webapp/.bridge/bridge-server.conf
```

Comment:
Is it worth spending some time figuring out why this is necessary? BridgeEX can pull configs from env vars no problem, and BridgeEX and BridgePF both use the same Config parser under the covers.

I also tested it locally, and my local Bridge server can pick up configs from env vars and JVM vars just fine.

If possible, I'd strongly prefer getting rid of this file. Otherwise, it's another moving part that can break and another piece of boilerplate we need to update every time we add a new config setting.